pkg/clusters/addons/kong/kong.go

```diff
@@ -128,28 +148,88 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		return fmt.Errorf("%s: %w", stderr.String(), err)
 	}
 
+	args := []string{"--kubeconfig", kubeconfig.Name(), "install", DefaultDeploymentName, "kong/kong"}
 	// configure for dbmode options
 	if a.dbmode == PostgreSQL {
-		a.deployArgs = append(a.deployArgs,
+		dbmodedeployArgs := []string{
 			"--set", "env.database=postgres",
 			"--set", "postgresql.enabled=true",
 			"--set", "postgresql.postgresqlUsername=kong",
 			"--set", "postgresql.postgresqlDatabase=kong",
 			"--set", "postgresql.service.port=5432",
-		)
+		}
+		a.deployArgs = append(a.deployArgs, dbmodedeployArgs...)
```

Comment:
We mentioned this yesterday, this fix is to apply minimal "--set" on top of https://github.com/Kong/kubernetes-testing-framework/blob/main/configs/enterprise-default.yaml, enterprise postgress only apply those minimum override using https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R151

To provide compatibility, all other deployment will merge whatever defaults args from 
https://github.com/Kong/kubernetes-testing-framework/pull/104/files#diff-3d68f23e86db256aa769ae3f812e898e238f059fd95d96ff32a452981f588070R230

There are some duplicates, but functionally, all repeat setting are the same, it works as expected.

Reply:
Helm is generally okay with this, though you need to keep in mind that the last value in the arg list wins:

```
11:06:52-0700 yagody $ helm template trk /tmp/symkong -f ~/src/charts/charts/kong/example-values/minimal-kong-hybrid-data.yaml --set env.foo=foo | grep -A1 FOO
        - name: KONG_FOO
          value: "foo"
11:07:01-0700 yagody $ helm template trk /tmp/symkong -f ~/src/charts/charts/kong/example-values/minimal-kong-hybrid-data.yaml --set env.foo=foo --set env.foo=bar | grep -A1 FOO
        - name: KONG_FOO
          value: "bar"
11:07:03-0700 yagody $ helm template trk /tmp/symkong -f ~/src/charts/charts/kong/example-values/minimal-kong-hybrid-data.yaml --set env.foo=foo --set env.foo=foo | grep -A1 FOO                                                                                                                                                          
        - name: KONG_FOO
          value: "foo"
```
So if they're not actually complete duplicates, this could invite unexpected behavior. Don't think there's anything particularly useful we could do in the test framework though.

Reply:
All the integration tests (on top of w/w/o enterprise dbless, w/w/o enterprise postgress) pass as expected in both KIC and KTF repo indicates there is not any conflict any where, all parameters are the same.

Reply:
removed duplicates 10c3442c7907272d9e48016067ae4eee87fe1c4c and 10c3442c7907272d9e48016067ae4eee87fe1c4c